libs/sdk-backend-bear/src/utils/api.ts

```diff
@@ -69,6 +69,23 @@ export const objRefToUri = async (
     return isUriRef(ref) ? ref.uri : authCall((sdk) => sdk.md.getObjectUri(workspace, ref.identifier));
 };
 
+/**
+ * Converts ObjRef instances to URIs. For UriRefs, it returns the URIs as they are,
+ * for IdentifierRef calls the backend and gets the URIs.
+ * @param refs - refs to convert
+ * @param workspace - workspace id to use
+ * @param authCall - call guard to perform API calls through
+ *
+ * @internal
+ */
+export const objRefsToUris = async (
+    refs: ObjRef[],
+    workspace: string,
+    authCall: BearAuthenticatedCallGuard,
+): Promise<Identifier[]> => {
+    return Promise.all(refs.map((ref) => objRefToUri(ref, workspace, authCall)));
+};
```

Comment:
please double check the endpoint used for the translation. I have a vague recollection that it is possible to do bulk translation.

Reply:
It is, it is located in `sdk.md.getUrisFromIdentifiers`. Please use this bulk method as it will be more performant.

Reply:
And maybe then use this in the objRefToIdentifier body so that the logic is unified.